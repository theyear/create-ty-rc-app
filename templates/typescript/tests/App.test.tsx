import { render } from "@testing-library/react";

import App from "../src/App";

describe("App", () => {
  it("snapshot", () => {
    const { asFragment } = render(<App />);
    expect(asFragment()).toMatchSnapshot();
  });

  it("content", async () => {
    const { baseElement } = render(<App />);

    expect(baseElement.textContent).toEqual("hello");
  });
});
