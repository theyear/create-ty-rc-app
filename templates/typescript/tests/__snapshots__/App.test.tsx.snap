// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snapshot 1`] = `
<DocumentFragment>
  <div>
    hello
  </div>
</DocumentFragment>
`;
